src/index.html

```diff
@@ -1,17 +1,176 @@
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="label" for="surname"
+          >Surname:
+          <input
+            minlength="1"
+            maxlength="50"
+            required
+            id="surname"
+            type="text"
+            autocomplete="off"
+            name="surname"
+          >
+        </label>
+        <label class="label" for="name"
+          >Name:
+          <input
+            minlength="1"
+            maxlength="50"
+            required
+            id="name"
+            type="text"
+            autocomplete="off"
+            name="name"
+          >
+        </label>
+        <label class="label" for="age"
+          >How old are You?
+          <input
+            min="12"
+            max="100"
+            required
+            id="age"
+            type="number"
+            name="age"
+            value="12"
+          >
+        </label>
+        <label class="label" for="birthdate"
+          >Full date of birth:
+          <input
+            required
+            id="birthdate"
+            type="date"
+            name="birthdate"
+          >
+        </label>
+        <label class="label" for="agreement"
+          >I accept the term of the agreement
+          <input
+            required
+            id="agreement"
+            type="checkbox"
+            name="agreement"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <label class="label" for="email"
+          >E-mail:
+          <input
+            required
+            id="email"
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="label" for="password"
+          >Password:
+          <input
+            required
+            id="password"
+            type="password"
+            name="password"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label class="label inlineBlock">Do you love cats?
+          <input
+            id="cats-yes"
+            type="radio"
+            name="cats"
+          >
+          <label class="inlineBlock" for="cats-yes">Yes</label>
+          <input
+            id="cats-no"
+            type="radio"
+            name="cats"
+          >
+          <label class="inlineBlock" for="cats-no">No</label>
+        </label>
+        <label class="label" for="color"
+          >What is your favorite color?
+          <input
+            id="color"
+            type="color"
+            name="color"
+          >
+        </label>
+        <label class="label" for="bedtime"
+          >What time do you go to bed?
+          <input id="bedtime"
+            type="time"
+            step="1"
+            name="bedtime"
+          >
```

Comment:
when multi-lined, each attribute should start at new line
```suggestion
          <input
            id="bedtime"
            type="time"
            step="1"
            name="bedtime"
          >
```